app/views/error_template.scala.html

```diff
@@ -14,16 +14,20 @@
  * limitations under the License.
  *@
 
-@this(govuk_wrapper: views.html.govuk_wrapper)
+@import views.Title
+@import views.html.components.gds._
 
-@(pageTitle: String, heading: String, message: String)(implicit request: Request[_], messages: Messages)
-
-@contentHeader = {
-  <h1>@heading</h1>
-}
+@this(govukLayout: gdsMainTemplate)
 
-@mainContent = {
-  <p>@message</p>
-}
+@(pageTitle: String, heading: String, message: String)(implicit request: Request[_], messages: Messages)
 
-@govuk_wrapper(title = pageTitle, contentHeader = Some(contentHeader), mainContent = mainContent)
+@govukLayout(
+ title = Title(pageTitle),
+ useCustomContentWidth = true) {
+ <div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+   <h1 class="govuk-heading-xl" >@heading</h1>
+   <p class="govuk-body">@message</p>
+  </div>
+ </div>
```

Comment:
Is this really the case? I wouldn't expect error page has to be wider.

Reply:
Yes that is the case.

Reply:
Well I mean if we want to maintain the same width as we do now we will have to go with that.